pywinauto/application.py

```diff
@@ -168,8 +168,8 @@ def __init__(self, search_criteria):
     def __call__(self, *args, **kwargs):
         """No __call__ so return a usefull error"""
         if "best_match" in self.criteria[-1]:
-            raise AttributeError(
-                "WindowSpecification class has no '{0}' method".
+            raise AttributeError("GUI element (wrapper) is not found " \
+                "or WindowSpecification has no '{0}' method (typo?)".

```

Comment:
I think it is better to highlight the wrong method name.
For example,
```python
"Neither GUI element (wrapper) nor wrapper method '{0}' were found (typo?)"
```

Reply:
Agree, will correct with adding some tests.